laravel5/app/Monomalpoly/Partie.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include('Case.php');
+
+class Partie{
+	private $listeCouleur = ['red','yellow','orange','blue','green','grey'];
+	private $listeNomCase = [];
+	private $listeJoueur;
+	private $plateau;
+	private $timer;
+
+	function Partie(){
+		$this->listeJoueur = [];
+		$this->timer = new Timer();
+
+		/* Initialisation du plateau (liste de cases)*/
+		for($i = 0 ;$i<$listeNomCase.length(); $i++){
+			/* Systeme pour l'affichage en x/y */
+
+			$this->plateau.add(new CaseDeJeux($listeNomCase[$i],$x,$y));
+		}
+	}
+
+	function joueurJoin(){
+		//On créé un nouveau joueur
+		$listeJoueur.add(new Joueur($_SESSION['id'],$_SESSION['nom'],$listeCouleur[$listeJoueur.length()]));
```

Comment:
This line is hardly readable. Split it into 2 lines.
Bonus: if you split it into 2 lines, you won't need the comment

Reply:
Using $_SESSION within a domain class is not a good practice because it increases the coupling between the class and the technology you're using. Pass needed information in the arguments of the function.